### PR TITLE
[MIG] purchase_minimum_amount: Migration to 13.0

### PR DESCRIPTION
**Pull request by albariera**
Standard migration. No real changes. 

Depends on #1123

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]
+1 technical review

File: purchase_minimum_amount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
```
Comment (MiquelRForgeFlow) [inline purchase_minimum_amount/__manifest__.py]:
```suggestion
# Copyright 2016 ForgeFlow S.L.
```

Could you do this everywhere please?

Comment (albariera) [reply purchase_minimum_amount/__manifest__.py]: Comment attended

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**